launcher/README.md

```diff
@@ -1,24 +1,40 @@
-The preflight updater GUI currently supports both PyQt4 and PyQt5. To
-enforce the use of PyQt5, set the environment variable SDW_UPDATER_QT to 5.
+# securedrop-workstation updater
 
-## Why support PyQt4 and PyQt5?
+The Updater ensures that the SecureDrop Workstation is up-to-date by checking for and applying any necessary VM updates, which may prompt a reboot.
 
-Qubes 4.0.3 uses an end-of-life Fedora template in dom0 (fedora-25). See
-rationale here:
+## Running the Updater
 
-https://www.qubes-os.org/doc/supported-versions/#note-on-dom0-and-eol
+Qubes 4.0.3 uses an end-of-life Fedora template in dom0 (fedora-25). See rationale here: https://www.qubes-os.org/doc/supported-releases/#note-on-dom0-and-eol.
 
-fedora-25 only includes PyQt4, which is why we have to support it for now.
-The next version of Qubes, Qubes 4.1, will include PyQt5 in dom0.
+To run the preflight updater:
+1. Open a `dom0` terminal
+2. Run `/opt/securedrop/launcher/sdw-launcher.py --skip-delta 0`
 
-## Installing PyQt4
+To run the notifier that pops up if `/proc/uptime` (how long the system has been on since its last restart) is greater than 30 seconds and `~/.securedrop_launcher/sdw-last-updated` (how long it's been since the Updater last ran) is greater than 5 days:
+1. Open a `dom0` terminal
+2. Run `/opt/securedrop/launcher/sdw-notify.py --skip-delta 0`
 
-PyQt4 is no longer maintained, and is best installed through system
-packages, e.g., https://packages.debian.org/buster/python3-pyqt4
+## Developer environment
 
-## Installing PyQt5
+The next version of Qubes will include PyQt5 in dom0, which is why we also support PyQt5.
 
-The recommended version of PyQt5 is included in the developer requirements
-for this project, which you can install via:
+### PyQt4 instructions
 
-pip install --require-hashes -r dev-requirements.txt
+To run the preflight updater outside of `dom0`:
+1. `cd securedrop-workstation/launcher`
+2. Make the launcher script executable: `chmod +x sdw-launcher.py`
```

Comment:
`git` lets us version that executable bit, which was already the case for `sdw-notify.py`. I can't think of a reason not to also set it for `sdw-launcher.py`, so I pushed up a small commit to do so. That should make this step unnecessary.